### PR TITLE
Feature: Add badge to Events nav item to boost engagement

### PR DESCRIPTION
**Pull request by krissy**
# Feature

Add badge to Events navigation item to boost engagement with the section. (Small changes can make a big diff - http://dev.venntro.com/2015/10/effects-of-small-changes/)

**Needs:** some UX opinions. Copy? Colours? Spacing?
## Tasks
- [x] Add badge to default nav
- [x] Add badge to responsive layout nav
### Default

screen shot 2016-04-07 at 17 37 19
### Responsive

screen shot 2016-04-07 at 17 36 55

### CODE REVIEW

**Comment by tanyapowell** [comment]
Will do [issue #29](https://github.com/empowerhack/empowerhack.github.io/issues/29) once you've been given the :+1: and this has been merged

**Comment by EChesters** [comment]
The pink box seems to be a CTA next to a CTA. I'm not sure this would boost engagement. Can I click it separately so it's one click straight through to signing up? It's not clear what I'm signing up to either, a meet-up group, a specific meet-up already arranged? 

We might need to go back to the drawing board with regards to the initial goal. Happy to help iron that out!

File: index.html [hunk 1]
```diff
@@ -42,7 +42,7 @@
 			 	<a href="#about">About</a>
 				<a href="#join">Join</a>
 			 	<a href="#projects">Projects</a>
-			 	<a href="#events">Events</a>
+			 	<a href="#events">Events<span class="label label-default">Sign up!</span></a>
```
Comment (neraks) [inline index.html:45]: Not a big deal, but would it be better to add a space between Events and Sign up?

Comment (eddiejaoude) [inline index.html:45]: Good spot @neraks . Same here too https://github.com/empowerhack/empowerhack.github.io/pull/26/files#diff-eacf331f0ffc35d4b482f1d15a887d3bR31

**Comment by eddiejaoude** [comment]
Can this PR be updated from `master`

**Pull request closed without merging**